mobile/src/full/java/org/openhab/habdroid/core/GcmIntentService.java

```diff
@@ -102,12 +103,12 @@ private void sendNotification(String msg, int notificationId) {
         NotificationCompat.Builder mBuilder =
                 new NotificationCompat.Builder(this)
                         .setSmallIcon(R.drawable.ic_openhab_appicon_24dp)
-                        .setContentTitle("openHAB")
+                        .setContentTitle(msg)
```

Comment:
The message title will now be the same as the message content. I'm not sure, if I like that :(